packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx

```diff
@@ -173,6 +177,7 @@ function WorklistDataGrid(props) {
 
     // Determine whether cell is a header or grid cell
     const WorklistCellTag = props.rowHeaderIndex === cellColumnIndex ? 'th' : 'td';
+    const height = props.rowHeight;
```

Comment:
We should destructure this as the other props are destructured as well:

https://github.com/cerner/terra-framework/blob/4374b14da5d55e67eafb27463670851bb4e6c4f6/packages/terra-worklist-data-grid/src/WorklistDataGrid.jsx#L76-L85

Reply:
Good point.  I will make this update.